tests/test_reporter.py

```diff
@@ -296,6 +294,29 @@ def send(_):
         assert reporter.queue.qsize() == 0, 'all spans drained'
         assert count[0] == 4, 'last span submitted in one extrac batch'
 
+    def test_force_flush(self, request):
+        block_for = 0
+        def send(_):
+            time.sleep(block_for)
```

Comment:
Any chance you'd be able to expand this w/ assertions to confirm expected spans are provided w/ an additional flush call?